addons/project_timesheet/project_timesheet_view.xml

```diff
@@ -113,7 +133,27 @@
                         <a name="open_timesheets" type="object">Timesheets</a>
                     </div>
                 </xpath>
+            </field>
+        </record>
 
+        <record id="view_task_kanban_inherited_progress" model="ir.ui.view">
+            <field name="name">project.task.timesheet.kanban.inherited.progress</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="project.view_task_kanban"/>
+            <field name="arch" type="xml">
+                <templates position="before">
+                    <field name="progress" />
+                    <field name="remaining_hours" />
+                </templates>
+                <div class="oe_kanban_bottom_left" position="inside">
+                   <t t-if="record.progress.raw_value &gt; 80 and record.progress.raw_value &lt; 100">
+                      <span t-att-class="'pull-right label label-' + (record.progress.raw_value &gt;= 99 ? 'danger': 'warning')"
+                            style="margin-left: 4px;"
+                            groups="project.group_time_work_estimation_tasks" >
+                           <t t-esc="_.str.sprintf('%d', record.remaining_hours.raw_value)"/>h
+                      </span>
```

Comment:
```html
                <templates position="before">
                    <field name="progress"/>
                </templates>
                <div class="oe_kanban_bottom_left" position="inside">
                   <t t-if="record.progress.raw_value &gt; 80 and record.progress.raw_value &lt; 100">
                           <span t-att-class="'pull-right label label-' + (record.progress.raw_value &gt;= 99 ? 'danger': 'warning')" style="margin-left: 4px;" groups="project.group_time_work_estimation_tasks" title="Remaining hours">
                                  <field name="remaining_hours" widget="float_time" />
                           </span>
                   </t>
               </div>
```

- minimalist style for margin & alignment, see with @qsm-odoo 
- JS fix in kanban_widgets.js:57: ``this.$el.text(formats.format_value(this.field.raw_value, this.format_descriptor));``